mne/epochs.py

```diff
@@ -779,10 +779,19 @@ class Epochs(_BaseEpochs, ToDataFrameMixin):
         Return Epochs object with a subset of epochs corresponding to an
         experimental condition as specified by 'name'.
 
+        If conditions are hierarchical (separated by '/', e.g. 'audio/left',
```

Comment:
+1 I don't see the hierarchy here. It's to me more a flat list of tags separated by /